src/test/kotlin/racingCar/CarTest.kt

```diff
@@ -0,0 +1,29 @@
+package racingCar
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+
+class CarTest {
+    private lateinit var car: Car
+
+    @BeforeEach
+    fun setUp() {
+        car = Car()
+    }
```

Comment:
알고 계실 것 같지만, BeforeEach 없이 아래와 같이 사용하셔도

테스트 별 환경이 격리되어 car가 다른 테스트에 영향을 받지 않습니다!

```suggestion
    private val car = Car()
```